npx-windows.md

```diff
@@ -0,0 +1,29 @@
+# Run NPX without errors on Windows
+
+Rearranging the `PATH` environment variable on Windows fixes common problems with **older versions** of NPX.
```

Comment:
If it's only a problem with older NPX versions, you can just upgrade npm instead, right?

Reply:
Yes, exactly.

Reply:
However, sometimes users do not have permission to use `-g` (to install global packages), and this solution may help them.

Reply:
> Edited: Actually, some users are still reporting this problem with newer NPX versions on NodeJS < 10.

Then the proper fix is to upgrade Node.js.

> However, sometimes users do not have permission to use -g (to install global packages), and this solution may help them.

Then the proper fix is to actually fix using `-g`, which is important for installing other packages globally too.

---

I'm gonna pass on this as there are better solutions to the problems mentioned here.

Reply:
Usually schools and some companies do not give users permission to upgrade NodeJS or NPM, since they receive non-admin accounts. The proposed solution is very useful for these cases. This use to happen all the time in lab classes with my students...